scripts/micro_benchmarks.py

```diff
@@ -160,10 +60,9 @@ def __dotnet_configuration(configuration: str) -> str:
 
     parser.add_argument(
         '-f', '--frameworks',
-        required=True,
+        required=False,
```

Comment:
big thanks for keeping the old feature working (I use it very frequently) 👍 

One funny side effect of this change is that if we run the script locally without specyfing the tfm or channel, we get a very strange python error:

```log
PS C:\Projects\performance> py .\scripts\benchmarks_ci.py
[2020/01/06 16:47:27][INFO] ----------------------------------------------
[2020/01/06 16:47:27][INFO] Initializing logger 2020-01-06 16:47:27.512296
[2020/01/06 16:47:27][INFO] ----------------------------------------------
Traceback (most recent call last):
  File ".\scripts\benchmarks_ci.py", line 262, in <module>
    __main(sys.argv[1:])
  File ".\scripts\benchmarks_ci.py", line 199, in __main
    .get_target_framework_monikers(args.frameworks)
  File "C:\Projects\performance\scripts\dotnet.py", line 79, in get_target_framework_monikers
    for framework in frameworks
TypeError: 'NoneType' object is not iterable
```

Previously it would complain about the lack of mandatory argument. Now we have two optional arguments. It would be nice to tell the user that tfm or channel has to be provided.

Reply:
Thanks for pointing it out! Will update this.